.github/workflows/sandbox-creating.yml

```
@@ -153,6 +153,9 @@ jobs:
           RESPONSE=$(curl -s -H "Authorization: Bearer $GITHUB_TOKEN" \
             -H "Accept: application/vnd.github+json" \
             "https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls?head=${GITHUB_REPOSITORY_OWNER}:${GITHUB_REF_NAME}")
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)